src/_components/ExploreManual.jsx

```diff
@@ -0,0 +1,109 @@
+export default function ExploreSection() {
+  return (
+    <section className="bg-background-grey py-12 md:py-24">
+      <div className="px-6 max-w-7xl mx-auto">
+        <h2 className="text-4xl font-bold text-green mb-12">
+          EXPLORE OUR SITE
+        </h2>
+
+        <div className="grid grid-cols-1 lg:grid-cols-5 gap-12 items-start">
+          <div className="lg:col-span-2 flex flex-col items-center lg:items-start">
+            <img
+              src="/assets/book.png"
+              alt="Manual Books"
+              className="w-full max-w-md h-auto mb-8"
+            />
+
+            <div className="flex items-center gap-6">
+              <h3 className="text-5xl font-bold text-primary">
```

Comment:
I think this is the wrong red. On "Read the Manual" can you also add an underline effect on Hover so people know its clickable